src/workerd/api/pyodide/pyodide.c++

```diff
@@ -8,20 +8,20 @@
 namespace workerd::api::pyodide {
 
 // singleton that owns bundle
-kj::TreeMap<kj::String, kj::Own<capnp::FlatArrayMessageReader>> bundleReaders;
+static kj::MutexGuarded<kj::TreeMap<kj::String, kj::Tuple<kj::Own<capnp::FlatArrayMessageReader>, jsg::Bundle::Reader>>> bundleReaders;
```

Comment:
`kj::Tuple` should only be used in template metaprogramming. Here, you should instead declare a two-element struct and use that. It results in code that's much easier to read, since the members are named.